cpp/src/arrow/ipc/options.h

```diff
@@ -128,6 +128,18 @@ struct ARROW_EXPORT IpcWriteOptions {
   static IpcWriteOptions Defaults();
 };
 
+/// \brief Alignment of data in memory
+/// Alignment values larger than 0 are taken directly as byte alignment value
+/// See util::EnsureAlignment(..., int64_t alignment, ...)
+enum class Alignment {
```

Comment:
This will be coerced to a size, let's make sure the type corresponds?
```suggestion
enum class Alignment : int64_t {
```